### DISCUSSION TITLE
Are there mature commercial productions using deno?

### DISCUSSION THREAD

**Discussion by shubulan** [denoland/deno#16962] [Q&A]
I'm going to use deno to develop some software. This software may be used by many people. I am concerned about the stability of the deno runtime
I noticed that many companies are paying attention to the project of deno https://github.com/denoland/deno/wiki # companies-interested-in-deno. and https://github.com/denoland/deno/discussions/15101 says Deno's been stable since v1.0.

But I want to know, at present, how is the stability of deno? Are there mature commercial projects using deno?

In addition, my software may run on mobile platforms, including android and ios. I see that deno can cross linux, windows and mac platforms. I would like to ask, is it feasible to migrate deno to a mobile platform?

**Comment by zyt123456789** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，您的邮件我已经收到，辛苦了！
